features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2

```diff
@@ -1496,4 +1496,10 @@
         <Exp>{{apim.testkey.validity.exp}}</Exp>
      </APITestKeyConfigs>
      {% endif %}}
+
+     {% if apim.api.creator.validation is defined %}
+     <APICreatorValidationConfigs>
+        <EnableOrgValidation>{{apim.api.creator.validation.enableOrganizationValidation}}</EnableOrgValidation>
+     </APICreatorValidationConfigs>
+     {% endif %}}
```

Comment:
Can we get this into choreo namespace. Simplify the name too
choreo.api_backend_url_validation.enable_org_validation